capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] Please make sure that this class inherits from the Decorator class in which the `@namebale` instance variable and the `correct_name` method are placed, so instead of `super` it's called for to use `@namebale.correct_name`